org.eclipse.lsp4e/plugin.xml

```diff
@@ -108,6 +108,11 @@
             id="org.eclipse.lsp4e.selectionRange.down"
             name="%command.selection.range.down.name">
       </command>
+      <command
+            categoryId="org.eclipse.lsp4e.category"
+            id="org.eclipse.lsp4e.typeHierarchy"
```

Comment:
I could not call the `Show Type Hierarchy` from context menu on text selection until i added:
```xml
         <command
               commandId="org.eclipse.lsp4e.typeHierarchy">
         </command>
```
after line 331 below `org.eclipse.lsp4e.openCallHierarchy`  under the menuContribution `popup:#TextEditorContext?before=org.eclipse.ui.genericeditor.findReferences`